apiary.apib

```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1164</td>
+            <td>Invalid result count value</td>
+        </tr>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1167</td>
+            <td>API key is missing</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1186</td>
+            <td>Invalid from date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1187</td>
+            <td>Invalid to date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1227</td>
+            <td>Invalid updated from date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        <tr>
+            <td>400 Bad Request</td>
+            <td>1228</td>
+            <td>Invalid updated to date. Allowed format is "MM/dd/yyyy"</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+## Get [/v1/review/report/count-by-rating/employee/{BusinessId}{?api_key}]
+Business review(s) from BirdEye platform can be fetched using following filter in request payload:
+
++ Parameters
 
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews Count By Driver [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+
+    curl -XPOST -H "Content-type: application/json" -d '{"businessNumbers": [145308764185002,145308769721320],"sourceAlias": ["google","citysearch"],"employees": [{"firstName": "String","lastName": "String","phone": "String"}],"fromDate": "04/01/2013","toDate": "05/3/2015"}' 'https://api.birdeye.com/resources/v1/review/report/count-by-rating/employee/145308769721320?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451'
+
+<center><b>Example Payload</b></center>
+
+    {
+     "businessNumbers": [
+       145308764185002,
+       145308769721320
+      ],
+     "sourceAlias": [
+     "google","citysearch"
+     ],
+     "employees": [
+       {
+      "firstName": "String",
+      "lastName": "String",
+      "phone": "String"
+       }
+     ],
+     "fromDate": "04/01/2013",
+     "toDate": "05/3/2015"
+   }
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Request Payload</caption>
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>fromDate</td>
+            <td>Review date (start) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>toDate</td>
+            <td>Review date (end) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>sourceAlias</td>
+            <td>Aggregation source name e.g. "google","citysearch" etc. Note: use "our_website" to filter reviews posted on Birdeye platform or use "direct_feedback" for direct feedback. </td>
+        </tr>
+         <tr>
```

Comment:
As mentioned for above API, describe the optional fields

Reply:
Done